app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,52 @@
+
+module Api::V1
+	class BooksController < ApplicationController
```

Comment:
We will need authentication and authorization here. For instance are we going to allow any user to update any book?